keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ArtifactSummary.kt

```diff
@@ -19,6 +21,7 @@ data class ArtifactVersionSummary(
   val environments: Set<ArtifactSummaryInEnvironment>
 )
 
+@JsonPropertyOrder(value = ["name", "version", "state"])
```

Comment:
@erikmunson This now ensures these properties show up first, in the specified order, in these summaries.